bitnami/kube-prometheus/Chart.yaml

```diff
@@ -21,11 +21,11 @@ dependencies:
 - condition: exporters.enabled,exporters.node-exporter.enabled
   name: node-exporter
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 3.x.x
+  version: 3.8.x
```

Comment:
was not 100% sure about wether this (and also the change below) is really needed, but on the other hand setting sample limit for this would just have no effect and could be confusing